.mailmap

```diff
@@ -278,13 +321,17 @@ Tarun Batra <tarun.batra00@gmail.com> Tarun <tarun.batra00@gmail.com>
 Ted Young <ted@radicaldesigns.org>
 Thomas Lee <thomas.lee@shinetech.com> <tom@tom-debian.sensis.com.au>
 Thomas Reggi <thomas@reggi.com>
+Tierney Cyren <hello@bnb.im> &! (bitandbang) <tieliaco@gmail.com>
+Tierney Cyren <hello@bnb.im> bitandbang <hello@bnb.im>
```

Comment:
@bnb is this correct? The first email address is linked to another account (TierneyC)